src/utils.js

```diff
@@ -112,3 +60,84 @@ export function buildClause (field, value, opts) {
 
   return Object.assign({}, mainClause, opts)
 }
+
+export function toBool (filters) {
+  const unwrapped = {
+    must: unwrap(filters.and),
+    should: unwrap(filters.or),
+    must_not: unwrap(filters.not),
+    minimum_should_match: filters.minimum_should_match
+  }
+
+  if (
+    filters.and.length === 1 &&
+    !unwrapped.should &&
+    !unwrapped.must_not
+  ) {
+    return unwrapped.must
+  }
+
+  const cleaned = {}
+
+  if (unwrapped.must) {
+    cleaned.must = unwrapped.must
+  }
+  if (unwrapped.should) {
+    cleaned.should = filters.or
```

Comment:
it depends: using `unwrapped.should` would be nicer, but changes the output of bodybuilder, so it would require a major bump. The current output that doesn't break the tests is `filters.or`.

Should I leave some comments here that this should be changed with the next major update of bodybuilder?